lib/tasks/blazegraph.rake

```diff
@@ -1,24 +1,24 @@
 require 'net/http'
 
 rails_env = ENV['RAILS_ENV'] || 'test'
-BLAZEGRAPH_HOME = ENV['CVM_BLAZEGRAPH_HOME'] || File.expand_path('./blazegraph')
-BLAZEGRAPH_DOWNLOAD_URL = ENV['CVM_BLAZEGRAPH_DOWNLOAD'] || "http://iweb.dl.sourceforge.net/project/bigdata/bigdata/2.1.0/blazegraph.jar"
-BLAZEGRAPH_URL = ENV['CVM_BLAZEGRAPH_URL'] || 'http://localhost:9999/blazegraph'
+BLAZEGRAPH_HOME = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_HOME'] || File.expand_path('./blazegraph')
+BLAZEGRAPH_DOWNLOAD_URL = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_DOWNLOAD'] || "http://iweb.dl.sourceforge.net/project/bigdata/bigdata/2.1.0/blazegraph.jar"
+BLAZEGRAPH_URL = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_URL'] || 'http://localhost:9999/blazegraph'
 BLAZEGRAPH_SPARQL = "#{BLAZEGRAPH_URL}/namespace/#{rails_env}/sparql"
-BLAZEGRAPH_CONFIG_LOG4J = ENV['CVM_BLAZEGRAPH_CONFIG_LOG4J'] || "./config/blazegraph/log4j.properties"
-BLAZEGRAPH_CONFIG_DEFAULT = ENV['CVM_BLAZEGRAPH_CONFIG_DEFAULT'] || "./config/blazegraph/blazegraph.properties"
-BLAZEGRAPH_CONFIG_PRODUCTION = ENV['CVM_BLAZEGRAPH_CONFIG_PRODUCTION'] || "./config/blazegraph/blazegraph.production.properties"
+BLAZEGRAPH_CONFIG_LOG4J = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_LOG4J'] || "./config/blazegraph/log4j.properties"
+BLAZEGRAPH_CONFIG_DEFAULT = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_DEFAULT'] || "./config/blazegraph/blazegraph.properties"
+BLAZEGRAPH_CONFIG_PRODUCTION = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_PRODUCTION'] || "./config/blazegraph/blazegraph.production.properties"
 
 namespace :scholars_archive do
```

Comment:
Yes. Check out lib/tasks/scholars_archive.rake on this branch to see the basic "setup" task that puts Fedora, Solr, and Blazegraph all into motion.